app/Console/Commands/ClearTrackCache.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/**
+ * Pony.fm - A community for pony fan music.
+ * Copyright (C) 2015 Peter Deltchev
```

Comment:
Put your name here and in the other copyright notices you added - this is your code. :smiley: